blog/_posts/2025-04-22-vm-generations-and-aks.md

```diff
@@ -0,0 +1,103 @@
+---
+title: "VM Generations and AKS"
+description: "Learn more about Generation 1 and Generation 2 VMs and what they offer, differences between them, upcoming Generation 1 VM retirements, and how to move your AKS workloads to Generation 2 VMs."
+date: 2025-04-22 # date is important. future dates will not be published
+authors: 
+   - Jack Jiang
+categories: 
+- operations
+---
+*This blog was co-authored by Ally Ford, Product Manager 2, AKS, Sarah Zhou, Product Manager, Azure Compute, and Jack Jiang, Product Manager, AKS*
+## What are Virtual Machine Generations?
+
```

Comment:
Might be nice to start this off with an intro sentence like "Did you know there are multiple generations of Azure VM's?" or something similar. I think it might help to set the context right at the start of the post